finite_state_sdk/__init__.py

```diff
@@ -2039,11 +2040,14 @@ def upload_file_for_binary_analysis(token, organization_context, test_id=None, f
         dict: The response from the GraphQL query, a completeMultipartUpload Object.
     """
     # To upload a file for Binary Analysis, you must use the generateMultiplePartUploadUrl mutation
-
     if not test_id:
         raise ValueError("Test Id is required")
     if not file_path:
         raise ValueError("File Path is required")
+    if chunk_size < 1024**2 * 5:
```

Comment:
Recommend moving these values to a constant at the beginning of this script or another module.